README.md

```diff
@@ -84,17 +85,19 @@ Want to contribute to the list? Read the [contributors guide](CONTRIBUTING.md)!
 * [Hive: Bitshares community](https://peakd.com/c/hive-120117/created) - Community dedicated for Bitshares on the Hive blockchain.
 * [Whaleshares#bitshares](https://whaleshares.io/created/bitshares) - #bitshares tagged posts on the WHALESHARES blockchain.
 * [Twitter@BitSharesGroup](https://twitter.com/bitsharesgroup) - Content managed by BitSharesGroup.org owner.
-* [Twitter@artcasa_fine](https://twitter.com/artcasa_fine) - Twitter account for the Bitshares Artcasa NFT Gallery
-* [Twitter@nftea_gallery](https://twitter.com/nftea_gallery) - Twitter account for the Bitshares NFTEA NFT Gallery
+* [Twitter@artcasa_fine](https://twitter.com/artcasa_fine) - Twitter account for the Bitshares Artcasa NFT Gallery.
+* [Twitter@nftea_gallery](https://twitter.com/nftea_gallery) - Twitter account for the Bitshares NFTEA NFT Gallery.
 * [Twitter#BitShares](https://twitter.com/search?q=%23BitShares) - #BitShares tagged on Twitter.
 * [Twitter#bitshares](https://twitter.com/search?q=%23bitshares) - #bitshares tagged on Twitter.
 * [Reddit#bitshares](https://reddit.com/r/bitshares) - Reddit sub-reddit for BitShares.
 * Telegram:
-  * [BitShares DAC](https://t.me/BitSharesDAC) - General discussions.
-  * [BitShares Group](https://t.me/BitSharesGroup) - Another group for general discussions.
-  * [BitShares News](https://t.me/BitSharesNews) - BitShares News channel
+  * [BitShares Group](https://t.me/BitSharesGroup) - Group for general discussions.
+  * [BitShares DAC](https://t.me/BitSharesDAC) - Another group for general discussions.
+  * [BitShares News](https://t.me/BitSharesNews) - BitShares News channel.
   * [BitShares Mobile](https://t.me/btsplusplus) - Focus on BitShares mobile app.
+  * [BitShares Mobile](https://t.me/BitSharesWallet) - Focus on BitShares reference wallet.
```

Comment:
Please update the name.

Reply:
Sorry; didn't get which name that you're referring to.

Reply:
> Sorry; didn't get which name that you're referring to.

The group name "BitShares Mobile".

Reply:
Done, thanks.